src/platform_impl/windows/drop_handler.rs

```diff
@@ -103,24 +108,35 @@ impl FileDropHandler {
     pub unsafe extern "system" fn DragOver(
         this: *mut IDropTarget,
         _grfKeyState: u32,
-        _pt: *const POINTL,
+        pt: POINTL,
         pdwEffect: *mut u32,
     ) -> HRESULT {
+        use crate::event::WindowEvent::DragOver;
         let drop_handler = unsafe { Self::from_interface(this) };
+        if drop_handler.enter_is_valid {
+            let mut pt = POINT { x: pt.x, y: pt.y };
+            unsafe {
+                ScreenToClient(drop_handler.window, &mut pt);
+            }
+            let position = PhysicalPosition::new(pt.x as f64, pt.y as f64);
+            drop_handler.send_event(Event::WindowEvent {
+                window_id: WindowId::from_raw(drop_handler.window as usize),
+                event: DragOver { position },
+            });
+        }
         unsafe {
             *pdwEffect = drop_handler.cursor_effect;
         }
-
         S_OK
     }
 
     pub unsafe extern "system" fn DragLeave(this: *mut IDropTarget) -> HRESULT {
-        use crate::event::WindowEvent::HoveredFileCancelled;
+        use crate::event::WindowEvent::DragLeave;
         let drop_handler = unsafe { Self::from_interface(this) };
-        if drop_handler.hovered_is_valid {
+        if drop_handler.enter_is_valid {
             drop_handler.send_event(Event::WindowEvent {
                 window_id: WindowId::from_raw(drop_handler.window as usize),
-                event: HoveredFileCancelled,
+                event: DragLeave,
             });
```

Comment:
I think it's wrong, if you had `Enter` you should have `Leave`, and you send `Enter` regardless. Sending `Enter` and then not `Leave` looks wrong.